doc/cross-compilation.xml

```diff
@@ -155,14 +157,21 @@
   <section>
     <title>Specifying Dependencies</title>
     <para>
-      As mentioned in the introduction to this chapter, one can think about a build time vs run time distinction whether cross-compiling or not.
-      In the case of cross-compilation, this corresponds with whether a derivation running on the native or foreign platform is produced.
-      An interesting thing to think about is how this corresponds with the three Autoconf platforms.
-      In the run-time case, the depending and depended-on package simply have matching build, host, and target platforms.
-      But in the build-time case, one can imagine "sliding" the platforms one over.
-      The depended-on package's host and target platforms (respectively) become the depending package's build and host platforms.
+      In this section we explore the relationship between both runtime and buildtime dependencies and the 3 Autoconf platforms.
+    </para>
+    <para>
+      A runtime dependency between 2 packages implies that between them both the host and target platforms match.
+      This is directly implied by the meaning of "host platform" and "runtime dependency":
+      The package dependency exists while both packages are runnign on a single host platform.
+    </para>
+    <para>
+      A build time dependency, however, implies a shift in platforms between the depending package and the depended-on package.
+      The meaning of a build time dependency is that to build the depending package we need to be able to run the depended-on's package.
+      The depending package's build platform is therefore equal to the depended-on package's host platform.
+      Analogously, the depending package's host platform is equal to the depended-on package's target platform.
+
+      In this manner, given the 3 platforms for one package, we can determine the three platforms for all its transitive build time dependencies.
```

Comment:
I like the way that blank line snuck in :), maybe this should be its own pargraph?

Reply:
Then we can remove the "build time" you added because this final paragraph will be talking about all kinds of dependencies, not just build time ones.